float64 left
float64 leftConf
float64 right
float64 rightConf
float64 div
float64 divConf
